integration/sawtooth_integration/tests/test_poet_smoke.py

```diff
@@ -85,17 +76,22 @@ def test_poet_smoke(self):
                 client.send_txns(increment)
 
         # wait for validators to catch up
-        time.sleep(CATCH_UP_TIME)
 
         self.assert_consensus()
 
     # if the validators aren't in consensus, wait and try again
     def assert_consensus(self):
-        try:
-            self._assert_consensus()
-        except AssertionError:
-            time.sleep(TRY_AGAIN_TIME)
-            self._assert_consensus()
+        time_out = time.time() + WAIT
+
+        while time.time() < time_out:
+            try:
+                self._assert_consensus()
+                return
+            except AssertionError:
+                LOGGER.info('Blocks not yet in concesus')
```

Comment:
Suggest rewriting this loop so the logging message has more information (ideally, a list of the block ids at that point and the time which has passed).

Reply:
added some logging to communicate how much time has passed. `_assert_consensus` already has logging to list the block ids